src/Fixer/StringNotation/NoBinaryStringFixer.php

```diff
@@ -31,7 +31,13 @@ final class NoBinaryStringFixer extends AbstractFixer
      */
     public function isCandidate(Tokens $tokens): bool
     {
-        return $tokens->isAnyTokenKindsFound([T_CONSTANT_ENCAPSED_STRING, T_START_HEREDOC]);
+        return $tokens->isAnyTokenKindsFound(
+            [
+                T_CONSTANT_ENCAPSED_STRING,
+                T_START_HEREDOC,
+                '"b',
```

Comment:
This should be

```suggestion
                'b"',
```


This is not detected by the added test because the fixer is still a candidate for the input code. Removing the first line of the test (i.e. `$fruit = "apple";`) fixes that.